tlatools/org.lamport.tlatools/src/util/UniqueString.java

```diff
@@ -339,7 +339,7 @@ public static UniqueString read(IDataInputStream dis, final Map<String, UniqueSt
         dis.readInt(); // skip, because invalid for the given internTbl
         final int slen = dis.readInt();
         final String str = dis.readString(slen);
-        return tbl.get(str);
```

Comment:
It's probably a good idea to leave a note as to why the Map gets ignored.

Reply:
Good idea. I actually found a way to (1) minimize tbl usage, and (2) also comment that minimal usage. See latest commit.